core-tests/src/test/java/org/mule/runtime/core/processor/chain/DefaultMessageProcessorChainTestCase.java

```diff
@@ -1027,13 +1036,15 @@ public Event process(Event event) throws MuleException {
 
   protected Event getTestEventUsingFlow(Object data) {
     Event event = mock(Event.class);
+    EventContext eventContext = DefaultEventContext.create(flow, "");
     InternalMessage message = InternalMessage.builder().payload(data).build();
     when(event.getFlowCallStack()).thenReturn(new DefaultFlowCallStack());
     when(event.getMessage()).thenReturn(message);
-    when(event.getExchangePattern()).thenReturn(exchangePattern);
+    when(event.getExchangePattern()).thenReturn(REQUEST_RESPONSE);
```

Comment:
can exchangePattern and syncronous attributes be removed already or are they being uses still?